modules/33acrossBidAdapter.js

```diff
@@ -93,12 +101,7 @@ function isBidRequestValid(bid) {
     return false;
   }
 
-  if ((typeof bid.params.site === 'undefined' || typeof bid.params.site.id === 'undefined') &&
-  (typeof bid.params.siteId === 'undefined')) {
-    return false;
-  }
-
-  if (typeof bid.params.productId === 'undefined') {
+  if (typeof bid.params.siteId === 'undefined' || typeof bid.params.productId === 'undefined') {
```

Comment:
Still not allowing test params to be passed via `site` object because:
1. The test bid feature makes this redundant
2. The test endpoint can now be set in `pbjs.config` object as follows:
     ```
      pbjs.setConfig({
            userSync: {
                iframeEnabled: true, 
                syncDelay: 3000
            },
            ttxSettings: {
                url: 'https://staging-ssc.33across.com/api/v1/hb',
                syncUrl: 'https://staging-de.tynt.com/deb/v2?m=xch'
            }
        });
     ```